docs/getting-started/install-endpoint.asciidoc

```diff
@@ -10,7 +10,7 @@ NOTE: To configure the Endpoint Security integration on the {agent}, you must ha
 [[security-before-you-begin]]
 == Before you begin
 
-If you're using macOS, some versions may require you to grant full disk access to different kernels, system extensions, or files. See <<sensor-full-disk-access>> for more information.
+If you're using macOS, some versions may require you to grant Full Disk Access to different kernels, system extensions, or files. Refer to <<deploy-elastic-endpoint, requirements for {elastic-endpoint}>> if you're installing the {elastic-endpoint} or <<endgame-sensor-full-disk-access, requirements for the Endgame sensor>> for more information.
```

Comment:
This page seems to be generic before it ends with this secotin https://security-docs_1501.docs-preview.app.elstc.co/guide/en/security/master/install-endpoint.html#enable-kernel-extension

Can we remove it and just say "for macOS, continue to [these](https://security-docs_1501.docs-preview.app.elstc.co/guide/en/security/master/deploy-elastic-endpoint.html) instructions to grant Endpoint the access it needs"